components/typography/Heading.js

```diff
@@ -0,0 +1,53 @@
+
+export default function Heading({
+    typeStyle = "heading-lg",
+    level = 'h2',
+    textColor = 'text-primary-800',
+    className,
+    children
+}) {
+    let classNames = ''
+    const Tag = `${level}`;
+    switch (typeStyle) {
+        case 'heading-xl': {
+            classNames = `font-heading text-2xl font-bold tracking-heading md:text-6xl leading-snug ${className || ''}`
+        }
```

Comment:
When you use `switch case` you need to remember about `break` statement. Here you have info why it's so important :) 

https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Statements/switch#what_happens_if_i_forgot_a_break

Reply:
Gotcha. Do I need `default` there at the end of the last case? If so, how should I handle that?

Reply:
Here you have `typeStyle` set to `heading-lg` default so you can write:

```js
default: {
  classNames = `font-heading text-2xl font-bold tracking-heading md:text-4xl leading-snug ${className || ''}`;
}
```

at last case in the `switch`. Here you don't need break because you call this as last case :)

Reply:
Ok check now